pyshelf/search_parser.py

```diff
@@ -18,19 +18,16 @@ def from_request(self, request_criteria):
         search_criteria = []
         sort_criteria = []
 
-        if isinstance(request_criteria["search"], list):
-            for search in request_criteria["search"]:
-                search_criteria.append(self._format_search_criteria(search))
-        else:
-            search_criteria.append(self._format_search_criteria(request_criteria["search"]))
+        for search in request_criteria["search"]:
+            search_criteria.append(self._format_search_criteria(search))
 
-        if isinstance(request_criteria["sort"], list):
+        if request_criteria.get("sort"):
```

Comment:
Yea that would require changing logic in the portal. At this point the "sort" key exists but can be None. So the get would return None and not [] since the key is there.